test/test_pebble.py

```diff
@@ -2775,6 +2946,16 @@ def test_exec_send_stdin(self):
         self.assertEqual(out, b'FOO\nBAR\n')
         self.assertEqual(err, b'')
 
+    def test_push_pull(self):
+        fname = os.path.join(tempfile.gettempdir(), 'pebbletest-{}'.format(uuid.uuid4()))
+        content = 'foo\nbar\nbaz-42'
+        self.client.push(fname, content)
+        with self.client.pull(fname) as f:
+            data = f.read()
+            self.assertEqual(data, content)
+        process = self.client.exec(['/bin/rm', fname])
```

Comment:
Guess we could just use `os.remove` here, as it's always on the same machine?

Reply:
changed.